src/workload.py

```diff
@@ -109,9 +109,7 @@ def exec(self, command: list[str], working_dir: str | None = None) -> str:
     )
     def alive(self) -> bool:
         try:
-            return bool(self.dashboards.services[self.SNAP_APP_SERVICE]["active"]) and bool(
-                self.dashboards.services[self.SNAP_EXPORTER_SERVICE]["active"]
-            )
+            return bool(self.dashboards.services[self.SNAP_APP_SERVICE]["active"])
```

Comment:
We had an ugly bug here. (@phvalguima is my witness :-) )

Reminder: OSD is a service that doesn't even come up (at all!) except in perfect conditions.
Meaning functional connection to Opensearch, etc. No, we do NOT even have a Status Page.
All we get is 
![image](https://github.com/user-attachments/assets/2caadf9b-5f32-405e-830a-ca326610babf)

The Prometheus Exproter process is very similar. It can't come up as long as there is no functional OSD available. It's failing right after re-start otherwise.

Now this resulted in an ugly, flaky behavior whenever OSD _was_ considered `active` on a snap level -- yet the service was not running (for example: due to missing Opensearch connection). Meaning that the first part of the condition above was `True` -- while the 2nd part was `True/False` ... depending on the moment.

![Screenshot from 2024-09-25 13-25-42](https://github.com/user-attachments/assets/733d4b2b-7695-4ba0-9c84-c6bfc586fb19)

Debugging underlying data structures (representing snap service state):

![Screenshot from 2024-09-25 13-25-54](https://github.com/user-attachments/assets/55ed485e-60bb-4733-b845-13906af9aa17)

The conclusion is this: we shouldn't check Prometheus Exporter state here. 

One proposal could be to do it in a separate check (I can propose a solution for that on a separte branch). In my opinion, as long as we are not providing similar safety measures on Openserach (where monitoring is a WAY more crucial), it may be best to keep comlexity low and trus automatic snap restarts to cover Prometheus Exporter failures.

Reply:
@juditnovak yes, I think we need to break the concepts: `alive()` here means the main service is up and running.